lib/plugins/popularity/helper.php

```diff
@@ -291,19 +291,19 @@ public function searchCountCallback(&$data, $base, $file, $type, $lvl, $opts)
     {
         // traverse
         if ($type == 'd') {
-            if ($data['dir_nest'] < $lvl) $data['dir_nest'] = $lvl;
-            $data['dir_count']++;
+            if ($data['dir_nest'] ?? 0 < $lvl) $data['dir_nest'] = $lvl;
```

Comment:
The $data array was already initialized by `initEmptySearchList()`... if I read the code correctly. How was it not defined here?